src/drawing/__tests__/DrawingManager-test.ts

```diff
@@ -0,0 +1,75 @@
+import ReactDOM from 'react-dom';
+import { createIntl } from 'react-intl';
+import DrawingManager from '../DrawingManager';
+import { createStore } from '../../store';
+import { Options } from '../../common/BaseManager';
+
+jest.mock('react-dom', () => ({
+    render: jest.fn(),
+    unmountComponentAtNode: jest.fn(),
+}));
+
+describe('DrawingManager', () => {
+    const intl = createIntl({ locale: 'en' });
+    const rootEl = document.createElement('div');
+    const getOptions = (options: Partial<Options> = {}): Options => ({
+        referenceEl: rootEl.querySelector('.reference') as HTMLElement,
+        ...options,
+    });
+    const getLayer = (): HTMLElement => rootEl.querySelector('[data-testid="ba-Layer--drawing"]') as HTMLElement;
+    const getWrapper = (options?: Partial<Options>): DrawingManager => new DrawingManager(getOptions(options));
+
+    beforeEach(() => {
+        rootEl.classList.add('root');
+        rootEl.innerHTML = '<div class="reference" />'; // referenceEl
+    });
+
+    describe('constructor', () => {
```

Comment:
Should we simplify by testing only what `DrawingManager` does rather than what the BaseManager does?